### DISCUSSION TITLE
SelectManyMenu  - Operation with the keyboard.

### DISCUSSION THREAD

**Discussion by hgfrey** [primefaces/community#736] [PrimeFaces] [answered]
How can I operate a SelectManyMenu element with the keyboard? In our application I can achieve "something" with Ctrl-Shift up or down. But I can't really see what's happening. In the current show case I can't achieve anything with the keyboard.

**Comment by melloware** [comment, marked as answer]
Fixed for 14.0.0: https://github.com/primefaces/primefaces/issues/1435
